SimplenoteWidgets/Models/SPManagedObject+CoreDataClass.swift

```diff
@@ -0,0 +1,7 @@
+import Foundation
+import CoreData
+
+@objc(SPManagedObject)
+public class SPManagedObject: NSManagedObject {
+
+}
```

Comment:
**Quesiton:**

Since Intents only requires Note, can we check what happens if the other (unused) "workaround NSMO Classes" are removed?

- `SPManagedObject`
- `Tag`
- `Settings`
- `Preferences`

Reply:
Intents will also need Tag, but can see if the others can be removed

Reply:
Seems to make Tag and Note compile we also need SPManagedObject, but the others can be removed